src/client/components/ContactForm/ContactForm.component.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ContactForm.styles.css';
+
+export default function ContactForm({user}) {
+    return (
+        <div className="infoBox">
+            <div className="wrapper">
+                <form className="form">
+                    <label className="title">CONTACT:</label>
+                    <label className="fullName">Full-name:  {
+                        user.fullName
+                    }</label>
+                    <label className="email">e-mail address:  {
+                        user.email
+                    }</label>
+                    <span className="formPaddingBottom"/>
```

Comment:
The span here is only to add a padding, you could add this property to the container of the labels.

Reply:
Thanks for the review, code adjusted as suggested.